### ISSUE TITLE
More keyboard shortcuts which follow conventions

### ISSUE THREAD

**Issue by machocam** [nextcloud/talk-desktop#919]
Keyboard shortcuts are especially important for desktop chat app. Jumping from chat to chat should be fast to be able to talk to many people/groups simultaneously.  

Instead of reinventing the wheel, I think that conventions should be used.  
In my experience, Telegram has the best keyboard navigation for desktop apps. I would simply copy what they have built in terms of keyboard navigation on desktop. 

### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Comment by ShGKme** [comment]
Yes, <kbd>Ctrl+PgUp</kbd>/<kbd>Ctrl+PgDown</kbd> or <kbd>Alt+Up</kbd>/<kbd>Alt+Down</kbd>  makes sense, wanted to add it a while ago.

The question is - do we want them on the Web as well (they conflict with the web-browser).

> In my experience, Telegram has the best keyboard navigation for desktop apps. I would simply copy what they have built in terms of keyboard navigation on desktop.

With all my love to Telegram's UX, cannot agree with this one. You cannot open a message menu, go to chat settings or right sidebar (chat info) via keyboard afak.

**Comment by nickvergessen** [comment]
> do we want them on the Web as well

 Not if they conflict

We should also respect this setting in the web and most likely add something a like in the Talk Desktop client settings:
![Image](https://github.com/user-attachments/assets/092f4832-8369-4aa8-8ab7-0dbba6167c40)

**Comment by machocam** [comment]
> With all my love to Telegram's UX, cannot agree with this one. You cannot open a message menu, go to chat settings or right sidebar (chat info) via keyboard afak.

Understood. For sure not everything is perfect.  
For me the most used keyboard shortcut is the cmd+k to jump from chat to chat -> their implementation of that is gold IMO. That is what I do most often and would argue most other people do. On the current version (unless I'm mistaken), i have to: 
1. Press escape to remove focus from chat window
2. Press cmd+f 
3. type the name
4. see the right chat come to the top
5. press enter twice to enter chat box
6. type message 

For some reason, even whatsapp doesn't do this. You can use cmd+f but after you type who you are searching for, enter doesn't change focus to that chat.   
That's the point of searching for someone with the keyboard, you use a hotkey, type 3 or 4 letters, get who you are looking for at the top, press enter and start typing your message ! I can jump to a group or an individual and type them a message on telegram in less than 2 seconds! no mouse involved.

**Comment by ShGKme** [comment]
> For me the most used keyboard shortcut is the cmd+k to jump from chat to chat

<kbd>cmd+k</kbd> is for inserting a link in the text. Do you mean <kbd>cmd+f</kbd>?

For the reference: https://usethekeyboard.com/telegram/

**Comment by nickvergessen** [comment]
https://usethekeyboard.com/microsoft-teams/ sounds like a good inspiration for our upcoming target :)

**Comment by machocam** [comment]
> cmd+k is for inserting a link in the text. Do you mean cmd+f?
> For the reference: https://usethekeyboard.com/telegram/

It seems that usethekeyboard is out of date.  Below an image from my desktop app taken today.  
If you have an account, give it a try - the flow is amazing. I can jump from chat to chat super fast using this quick search of theirs. 
![Image](https://github.com/user-attachments/assets/5473ae90-d820-4e2e-8e83-51f0adb19f4e)

**Comment by machocam** [comment]
A keyboard shortcut for replying to a message would be great too!  
I have to stop adding things here....

**Comment by fraenki** [comment]
A keyboard shortcut to edit the last message would be much appreciated. That's the main thing I'm currently missing from Talk Desktop. 😊

**Comment by nickvergessen** [comment]
> A keyboard shortcut to edit the last message would be much appreciated.

Try <kbd>Ctrl</kbd> + <kbd>ArrowUp</kbd>

**Comment by fraenki** [comment]
> Try Ctrl + ArrowUp

Woah, thanks! 🥳  I totally missed the overview of these shortcuts in the Talk settings. 🤦

**Comment by Fuseteam** [comment]
adding here for visibility:
i believe up as opposed to control+up is more conventional

it works like this in at least the following applications:
- telegram
- element
- cinny
- discord
- teams
- twitch chat
